data/migrations/20220121194245_role_tickets.js

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
It looks like this file was accidentally committed -- a prior branch, `feature/role-ticket-table`, already created this migration. It looks like you added it in to make sure your seeds were working (which is great!), but I don't think a re-upload is good, because it will entail sorting an unnecessary merge conflict down the line.